test/Microsoft.AspNet.Mvc.Core.Test/Rendering/HtmlHelperCheckboxTest.cs

```diff
@@ -262,15 +292,20 @@ public void CheckBoxForGeneratesUnobtrusiveValidationAttributes()
             Assert.Equal(expected, html.ToString());
         }
 
-        [Fact]
-        public void CheckBoxForUsesModelStateAttemptedValue()
+        [Theory]
+        [InlineData("false", "")]
+        [InlineData("true", "checked=\"checked\" ")]
+        public void CheckBoxFor_UsesModelStateAttemptedValue(string attemptedValue, string expectedChecked)
         {
             // Arrange
-            var expected = @"<input id=""Property1"" name=""Property1"" type=""checkbox"" value=""true"" />" +
+            var expected = @"<input {0}id=""Property1"" name=""Property1"" type=""checkbox"" value=""true"" />" +
                            @"<input name=""Property1"" type=""hidden"" value=""false"" />";
+            expected = string.Format(expected, expectedChecked);
```

Comment:
a) this is test code :smiley_cat: 
b) culture doesn't affect how `string` values get formatted